app/assets/stylesheets/_landing-page.scss

```diff
@@ -115,32 +115,41 @@
     }
   }
 
-  .tile-image-container {
+  .video-wrapper {
     border-radius: $base-border-radius;
-    height: 180px;
-    margin-bottom: $small-spacing;
+    display: inline-block;
+    margin-bottom: $base-spacing;
     overflow: hidden;
+    position: relative;
+    width: 100%;
 
-    a {
+    &::after {
+      content: "";
       display: block;
-      position: relative;
-
-      &:hover::after {
-        background: transparentize($black, 0.4);
-      }
+      padding-top: 61%;
     }
+  }
 
-    img {
-      border-radius: $base-border-radius;
-      width: 100%;
+  .video-thumbnail {
+    @include position(absolute, 0);
+    background: image-url("video-thumbnail.jpg") center center / cover no-repeat;
```

Comment:
Yeah, if you are using the shorthand `background` property, you have to separate the `background-position` and the `background-size` properties using `/`